Bencodex/Types/List.cs

```diff
@@ -54,6 +54,8 @@ public string Inspection
 
         IValue IReadOnlyList<IValue>.this[int index] => Value[index];
```

Comment:
`List` actually has had the `[int]` indexer here, except it's an [explicit interface implementation](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation).  Apparently now this line is redundant and unnecessary so it's okay to remove.

Reply:
Yes, you're right, I removed the line. 9577990